zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/PublishNotification.java

```diff
@@ -38,13 +38,12 @@ public class PublishNotification {
     }
 
     public PublishNotification(Collection collection, List<String> urisToUpdate, List<ContentDetail> urisToDelete) {
-        // Delay the clearing of the cache after publish to minimise load on the server while publishing.
-        Date clearCacheDate = new DateTime(collection.getDescription().getPublishDate())
-                .plusSeconds(Configuration.getSecondsToCacheAfterScheduledPublish()).toDate();
-
         if (Configuration.isLegacyCacheAPIEnabled()) {
-            this.legacyCacheApiPayloads = new LegacyCacheApiPayloadBuilder.Builder().collection(collection).publishDate(clearCacheDate).build().getPayloads();
+            this.legacyCacheApiPayloads = new LegacyCacheApiPayloadBuilder.Builder().collection(collection).build().getPayloads();
```

Comment:
Should we log publishdate here ?

Reply:
We're already logging each individual payload when we send them to the API, and that includes the publishing date, so I think that covers it.